wolftpm/tpm2_wrap.h

```diff
@@ -25,15 +25,81 @@
 
 #include <wolftpm/tpm2.h>
 
+typedef struct WOLFTPM2_HANDLE {
+    TPM_HANDLE      hndl;
+    TPM2B_AUTH      auth;
+} WOLFTPM2_HANDLE;
+
+typedef struct WOLFTPM2_SESSION {
+    WOLFTPM2_HANDLE handle;
+    TPM2B_NONCE     nonceTPM;
+} WOLFTPM2_SESSION;
+
+typedef struct WOLFTPM2_KEY {
+    WOLFTPM2_HANDLE   handle;
+    TPM2B_PRIVATE     private;
```

Comment:
private or public will cause problems if trying to build with g++

Reply:
please also do a check through the for other locations of public/private

Reply:
Good find! Making fix for that.